pkgs/by-name/bu/burpsuite/package.nix

```diff
@@ -109,7 +109,7 @@ buildFHSEnv {
     hydraPlatforms = [ ];
     maintainers = with maintainers; [
       bennofs
-      fab
+      fabaff
```

Comment:
This is my GitHub handle. GitHub handle and maintainer name don't have to be identical.

Needs to be reverted.

Reply:
Sorry 🫣